schema/workflow.json

```diff
@@ -50,6 +68,24 @@
             "items": {
                 "type": "object"
             }
+        },
+        "isOutdated" : {
+            "type": "boolean"
+        },
+        "history": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "type": "string"
+                    },
+                    "revision": {
+                        "type": "number"
+                    }
+                },
+                "required": ["id", "revision"]
+            }
```

Comment:
We should define history in a separate object (maybe `system` folder?) and reference here